src/main/io/displayport_max7456.c

```diff
@@ -193,15 +201,10 @@ static const displayPortVTable_t max7456VTable = {
 
 displayPort_t *max7456DisplayPortInit(const vcdProfile_t *vcdProfile)
```

Comment:
If the init is done elsewhere maybe consider altering the function naming to avoid confusion later on